rest/route/project_test.go

```diff
@@ -1148,7 +1148,7 @@ func TestDeleteProject(t *testing.T) {
 	require.NoError(t, badProject.Insert())
 	pdh.projectName = badProject.Id
 	resp = pdh.Run(ctx)
-	assert.Equal(t, http.StatusBadRequest, resp.Status())
+	assert.Equal(t, http.StatusOK, resp.Status())
```

Comment:
nit: can you update the ID / variable name here, "bad_project" seems inaccurate now

Reply:
Yes!